WordPress/Classes/Extensions/UIImageView+SiteIcon.swift

```diff
@@ -92,13 +94,9 @@ extension UIImageView {
             case .failure(let error):
                 if case .requestCancelled = (error as? AFIError) {
                     // Do not log intentionally cancelled requests as errors.
-                } else if case let Alamofire.AFError.responseValidationFailed(reason) = error,
-                          case let Alamofire.AFError.ResponseValidationFailureReason.unacceptableStatusCode(code) = reason,
-                          code == 404 {
-                    // Do not log 404 errors since they are expected for site icons
-                }
-                else {
+                } else {
                     WordPressAppDelegate.crashLogging?.logError(error)
+                    DDLogError(error.localizedDescription)
```

Comment:
In the rest of the code, the call to `logError` has been replaced with `DDLogError()`. Here, it runs alongside it. Is this intentional?

Reply:
This is not intentional... I forgot to clean up the original line, good catch!

Reply:
Removed